src/ReadVTK.jl

```diff
@@ -466,7 +467,9 @@ Retrieve a lightweight `{VTKData` object with the coordinate data of the given V
 
 See also: [`PVTKData`](@ref), [`get_point_data`](@ref),  [`get_cell_data`](@ref)
 """
-get_coordinate_data(pvtk_file::PVTKFile) = get_data_section.(pvtk_file.vtk_files, "Coordinates")
+function get_coordinate_data(pvtk_file::PVTKFile)
+  get_data_section.(pvtk_file.vtk_files, "Coordinates")
```

Comment:
```suggestion
  return get_data_section.(pvtk_file.vtk_files, "Coordinates")
```
Is there a way to automatically add `return` of the function is reformatted this way?

Reply:
Yes - done